src/views/filters/FiltersView.tsx

```diff
@@ -8,41 +8,40 @@ import TableCell, { TableCellProps } from "@material-ui/core/TableCell";
 import TableContainer from "@material-ui/core/TableContainer";
 import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
+import DeleteIcon from "@material-ui/icons/Delete";
+import Spinning from "../../components/spinning";
+import VisibilityIcon from "@material-ui/icons/Visibility";
 
 import Header from "../../components/header/Header";
 
 import "../../components/incidenttable/incidenttable.css";
 import FilterBuilder from "../../components/filterbuilder/FilterBuilder";
 import { withRouter } from "react-router-dom";
-import api, { IncidentMetadata, Filter, FilterDefinition, SourceSystem, FilterSuccessResponse } from "../../api";
+import api, { IncidentMetadata, Filter, FilterDefinition, FilterSuccessResponse, SourceSystem } from "../../api";
+import { toMapNum, toMapStr, pkGetter } from "../../utils";
+import TagSelector, { Tag, originalToTag } from "../../components/tagselector";
+import FilteredIncidentTable from "../../components/incidenttable/FilteredIncidentTable";
 
-import { pkGetter, toMap } from "../../utils";
-
-import IncidentsPreview from "../../components/incidentspreview/IncidentsPreview";
+import "../../components/incidenttable/incidenttable.css";
 
+import { FiltersContext, FiltersContextType, DEFAULT_FILTERS_CONTEXT_VALUE } from "../../components/filters/contexts";
```

Comment:
The "../../components/filters/contexts" file is not included in this commit and does not exist on master.